tests/test_env_yml.py

```diff
@@ -0,0 +1,23 @@
+"""
+Test if the environment.yml is empty or it constains other data structure than a dictionary
+"""
+import os
+import sys
+import pytest
+from repo2docker import buildpacks
+
+#@pytest.mark.xfail(reason="Not yet implemented")
+def test_empty_env_yml(tmpdir):
+    tmpdir.chdir()
+    p = tmpdir.join("environment.yml")
+    bp = buildpacks.CondaBuildPack()
+    with pytest.raises(Exception):
+        py_ver = bp.python_version()
```

Comment:
python_version is a `@property`, not a method, so this should be `py_ver = bp.python_version`